packages/lesswrong/lib/collections/posts/sortOrderOptions.tsx

```diff
@@ -0,0 +1,13 @@
+export interface SettingsOption {
```

Comment:
These declarations needed to be moved out of posts/schema.ts to avoid a dependency cycle.